src/server/pfs/server/driver.go

```diff
@@ -649,6 +649,7 @@ func (d *driver) createProjectInTransaction(txnCtx *txncontext.TransactionContex
 	if err := projects.Create(pfsdb.ProjectKey(req.Project), &pfs.ProjectInfo{
 		Project:     req.Project,
 		Description: req.Description,
+		CreatedAt:   types.TimestampNow(),
```

Comment:
I don't see an import statement in the diff so I assume this `types` package is the right way to do this.